src/giskardpy/plugin_post_processing.py

```diff
@@ -0,0 +1,73 @@
+import giskardpy.identifier as identifier
+from giskardpy.plugin import GiskardBehavior
+from giskardpy.exceptions import InsolvableException
+from py_trees import Status
+import numpy as np
+from giskardpy.plugin_interrupts import detect_wiggling
+
+
+class PostProcessing(GiskardBehavior):
```

Comment:
I was thinking of `PostProcessing` plugin to be just a sequence with two children. One does this goal constraints satified check, and the other is just the same wiggle detection plugin from before.